### ISSUE TITLE
Add trend arrow to area_profiles in fingertipscharts package

### ISSUE THREAD

**Issue by d93jwy** [PublicHealthEngland/fingertipscharts#13]
It would be useful to be able to add trend arrows to spine charts created using function area_profiles in the fingertipscharts package, to mirror the ones on fingertips (https://fingertips.phe.org.uk/profile/child-health-profiles/data#page/1/gid/1938133228/pat/6/par/E12000001/ati/202/are/E06000001)

**Comment by sebastian-fox** [comment]
Thanks for raising this @d93jwy - I'll look to incorporate this.
Here is a current example work around (for my benefit too for when implementing the change):

```
library(fingertipscharts)
library(dplyr)
library(ggplot2)

df <- create_test_data() %>%
  mutate(Value = case_when(
    grepl("2$|4$|6$", IndicatorName) ~ round(Value,1),
    TRUE ~ round(Value, 0)),
    trend = sample(c("up", "down", "none"), 666, replace = TRUE),
    arrow = case_when(
      trend == "up" ~ '\u2191',
      trend == "none" ~ "\u2192",
      trend == "down" ~ "\u2193",
      TRUE ~ "ERROR"
    )) 
full_p <- area_profiles(df,
                        value = Value,
                        count = Count,
                        area_code = AreaCode,
                        local_area_code = "AC122",
                        indicator = IndicatorName,
                        timeperiod = Timeperiod,
                        polarity = Polarity,
                        significance = Significance,
                        area_type = AreaType,
                        median_line_area_code = "C001",
                        comparator_area_code = "PAC12",
                        datatable = TRUE,
                        relative_domain_text_size = 0.75,
                        relative_text_size = 1.2,
                        bar_width = 0.68,
                        indicator_label_nudgex = -0.1,
                        show_dividers = "outer",
                        header_positions = c(-1, -0.44, -0.35, -0.25,
                                             -0.15, -0.05, 1.08),
                        dps = NA)

cols <- c('up' = '#92D050',
          'none' = '#FFC000',
          'down'='#C00000')
full_p +
  geom_text(data = df[df$AreaCode == "AC122",],
            aes(y = -0.7, label = arrow, colour = trend)) +
  scale_colour_manual(values = cols)
```

**Comment by sebastian-fox** [comment]
Fixed in fd70021e69115fa8141070dc7d54fd42a724a86a
